transform_inline.ml

```diff
@@ -107,8 +111,16 @@ let inline ({main; functions} as orig_prog : program) : program option =
      `caller` labels *)
   let replace_labels caller ({formals; instrs} as callee) =
     let used_labels = extract_labels caller.instrs in
-    let replace l = fresh used_labels l in
+    let callee_labels = LabelSet.elements (extract_labels callee.instrs) in
+    let replacements =
```

Comment:
Given that this pattern is repeating itself both on line 60 and 116, we can factor it out as a function called `fresh_many` (just like `subst` is generalized to `subst_many`) in `edit.ml`.